translations/.tx/config

```diff
@@ -2,52 +2,15 @@
 host = https://www.transifex.com
 
 [owncloud.client]
-host = https://www.transifex.com
 source_file = client_en.ts
 source_lang = en
-
-# simple one-to-one language mappings
-trans.ca = client_ca.ts
-trans.el = client_el.ts
-trans.es = client_es.ts
-trans.es_AR = client_es_AR.ts
-trans.eu = client_eu.ts
-trans.fa = client_fa.ts
-trans.fr = client_fr.ts
-trans.gl = client_gl.ts
-trans.it = client_it.ts
-trans.nb_NO = client_nb_NO.ts
-trans.nl = client_nl.ts
-trans.pl = client_pl.ts
-trans.pt_BR = client_pt_BR.ts
-trans.ru = client_ru.ts
-trans.sl = client_sl.ts
-trans.sv = client_sv.ts
-trans.sr = client_sr.ts
-trans.tr = client_tr.ts
-trans.uk = client_uk.ts
-trans.zh_TW = client_zh_TW.ts
-trans.zh_CN = client_zh_CN.ts
-
-# special handling below
+file_filter = client_<lang>.ts
+type = TS
 
 # de_DE holds the formal translation which we want as default
-trans.de_DE = client_de.ts
-
-# choose one of the given translations on transifex as default
-trans.pt_PT = client_pt.ts
-
-# choose a special language as more generic default
-trans.cs_CZ = client_cs.ts
-trans.et_EE = client_et.ts
-trans.fi_FI = client_fi.ts
-trans.ja_JP = client_ja.ts
-trans.hu_HU = client_hu.ts
-trans.sk_SK = client_sk.ts
-trans.th_TH = client_th.ts
+lang_map = de_DE: de, pt_PT: pt, cs_CZ: cs, et_EE: et, fi_FI: fi, ja_JP: ja, hu_HU: hu, sk_SK: client_sk, th_TH: th
```

Comment:
@fmoc is this mapping still needed/up to date with your recent changes and if so maybe we can still get rid of it?

Reply:
is there no multiline notation of this mapping?

would be nicer to review diffs later on .. but okay anyway